src/api/.keystone/config.js

```diff
@@ -95,6 +97,23 @@ var ACCOUNT = {
       const future = new Date(now.setMilliseconds(milliseconds));
       return future;
     }
+  },
+  JWT: {
+    KEY: {
+      SIGNATURE: String(process.env.JWT_SIGNING_KEY),
+      ENCODING: "base64",
+      STRING_ENCODING: "ascii"
+    },
+    TOKEN: {
+      EXPIRY: "8h",
+      ALGORITHM: "RS256"
+    },
+    SESSION_EXPIRY: () => {
+      const now = new Date();
+      const hours = 8;
+      const future = new Date(now.getTime() + hours * 60 * 60 * 1e3);
```

Comment:
Can the number of second be loaded from a constant please?

Reply:
This file is auto generated by keystone, the actual code we write is in `api/constants.ts` - same comment there, have addressed